pajbot/modules/vip_refresh.py

```diff
@@ -0,0 +1,151 @@
+import logging
+
+from sqlalchemy import text
+
+from pajbot.managers.db import DBManager
+from pajbot.managers.handler import HandlerManager
+from pajbot.managers.schedule import ScheduleManager
+from pajbot.models.command import Command, CommandExample
+from pajbot.modules import BaseModule, ModuleType
+from pajbot.utils import time_method
+
+log = logging.getLogger(__name__)
+
+
+class VIPRefreshModule(BaseModule):
+    ID = __name__.split(".")[-1]
+    NAME = "VIP refresh"
+    DESCRIPTION = "Regularly updates data about who is VIP"
+    ENABLED_DEFAULT = True
+    HIDDEN = True
+    MODULE_TYPE = ModuleType.TYPE_ALWAYS_ENABLED
+    CATEGORY = "Internal"
+
+    UPDATE_INTERVAL = 10  # minutes
+
+    def __init__(self, bot):
+        super().__init__(bot)
+        self.scheduled_job = None
+
+    def update_vip_cmd(self, bot, source, **rest):
+        # TODO if you wanted to improve this: Provide the user with feedback
+        #   whether the update succeeded, and if yes, how many users were updated
+        bot.whisper(source, "Reloading list of VIPs...")
+        self._update_vips()
+
+    def _update_vips(self):
+        self.bot.privmsg("/vips")
+
+    @staticmethod
+    def _parse_pubnotice_for_vips(msg_id, message):
+        if msg_id == "no_vips":
+            return []
+
+        if msg_id == "vips_success":
+            if message.startswith("The VIPs of this channel are: "):
+                message = message[30:]  # 36 = length of the above "prefix"
```

Comment:
mismatch, comment says 36 and code says 30

Reply:
👍